sandbox/public/index.html

```diff
@@ -52,7 +52,7 @@
           });
           tag.nonce = "%REACT_APP_NONCE%";
           tag.src = src;
-          document.head.append(tag);
+          document.head.appendChild(tag);
```

Comment:
Unrelated, but the sandbox would throw an error when running in Internet Explorer.